spec/person_spec.rb

```diff
@@ -0,0 +1,23 @@
+require './src/classes/person'
+
+describe Person do
+  context 'before each test in the person class' do
+    before(:each) do
+      @person = Person.new(18, 'Mwape', parent_permission: true)
+    end
+
+    describe '#new' do
```

Comment:
- [ ]  For the _Person_ class make sure to check if `can_use_services` equals true or false.
- [ ] Also, make sure to test both _positive_ and _negative_ cases.

Reply:
Done!